app/gui2/src/components/GraphEditor/GraphVisualization.vue

```diff
@@ -294,6 +296,7 @@ provideVisualizationConfig({
   hide: () => emit('update:visible', false),
   updateType: (id) => emit('update:id', id),
   addNode: (pos) => emit('addNode', pos),
+  createNode: (pattern) => emit('createNode', pattern),
```

Comment:
What is `addNode`? The name does not suggest any difference from `createNode` - perhaps those should be just merged?